tests/main/interfaces-snap-interfaces-requests-control/task.yaml

```diff
@@ -0,0 +1,90 @@
+summary: Verify that snap-interfaces-requests-control grants read-access to relevant endpoints
+details: |
+    The snap-interfaces-requests-control interface grants access to prompting clients
+    (e.g. prompting-client) to a subset of snapd's API endpoints needed
+    to view and manage request prompts and request rules.
+
+    Specifically:
+      - /v2/notices: to read change-update and refresh-inhibit notices
+      - /v2/system-info: to check whether prompting is supported/enabled
+      # TODO: - /v2/interfaces/requests/prompts: to receive and reply to request prompts
+      # TODO: - /v2/interfaces/requests/rules: to view and manage request rules
+
+environment:
+    # not all terminals support UTF-8, but Python tries to be smart and attempts
+    # to guess the encoding as if the output would go to the terminal, but in
+    # fact all the test does is pipe the output to jq
+    PYTHONIOENCODING: utf-8
```

Comment:
Perhaps setting locale to C.UTF8 would be more general?

Reply:
I don't see `C.UTF8` as an option here: https://docs.python.org/3/library/codecs.html#standard-encodings

Here's what I looked at:
1. https://docs.python.org/3/using/cmdline.html#envvar-PYTHONIOENCODING
2. https://docs.python.org/3/library/stdtypes.html#str.encode
3. https://docs.python.org/3/library/codecs.html#standard-encodings

Reply:
Default is `utf-8` though, so here we're forcing it so it doesn't try to be clever.